media/CircleGraph/index.js

```diff
@@ -51,6 +51,7 @@ const vscode = acquireVsCodeApi();
 const viewMode = {
     viewer: 0,
     selector: 1,
+    visq: 2,
```

Comment:
Q. Could you add any documentation (README or simple comment) explaning about what 'visq' is..?

Reply:
Thanks for the comment :)
I'll add comments in another PR.